static/src/styles/PostCard.styl

```diff
@@ -1,20 +1,75 @@
 .PostCard {
-  background: $Color__bright1;
-  padding: grid(2);
+  background: $Color__bright2;
+  background-size: cover;
   border-radius: $Layout__normalBorderRadius;
   min-height: grid(50);
   color: $Color__lightText;
+  position: relative;
+  overflow: hidden;
+  z-index: 0;
+  cursor: pointer;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
 }
 
-.PostCard__label {
+.PostCard__photoBackground {
+  display: block;
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  opacity: 0.3;
+  background-repeat: no-repeat;
+  background-position: 50% 0;
+  -ms-background-size: cover;
+  -o-background-size: cover;
+  -moz-background-size: cover;
+  -webkit-background-size: cover;
```

Comment:
Could we get some [Autoprefixer](https://github.com/postcss/autoprefixer) in here? Relying on humans to remember vendor prefixes is probably going to trip us up at some point.